### ISSUE TITLE
still maintained?

### ISSUE THREAD

**Issue by johnmmcgee** [4O4/mtasa-docker#7]
I haven't seen a build in some times.  Is this still maintained?

**Comment by 4O4** [comment]
Yes, it's maintained. I do public builds upon request when I'm not actively using MTA server myself and not monitoring the stable releases.

[1.6.0-22511-v1](https://hub.docker.com/layers/notfound/mtasa-server/1.6.0-22511-v1/images/sha256-962756907dbea7670aa0e321edad8767f8fd9c003d40ca14da133de15f0712dc?context=explore) has been published
